--- colors/jellybeans.vim	2013-03-07 00:08:15.000000000 -0800
+++ colors/jellybeans.vim	2013-03-07 00:11:09.000000000 -0800
@@ -317,7 +317,7 @@
 
 call s:X("LineNr","605958","151515","none",s:termBlack,"")
 call s:X("Comment","888888","","italic","Grey","")
-call s:X("Todo","808080","","bold","White",s:termBlack)
+call s:X("Todo","101010","fad07a","bold","White",s:termBlack)
 
 call s:X("StatusLine","000000","dddddd","italic","","White")
 call s:X("StatusLineNC","ffffff","403c41","italic","White","Black")
